src/indexes.rs

```diff
@@ -899,14 +899,12 @@ impl Index {
     pub async fn add_or_update<T: Serialize>(
         &self,
         documents: &[T],
-        primary_key: Option<impl AsRef<str>>,
+        primary_key: Option<&str>,
```

Comment:
Hey, I’m not sure to understand what you changed here?

We could already send an option with the old API right?
> primary_key: Option<impl AsRef<str>>,